GitUI/UserControls/FileStatusList.cs

```diff
@@ -1103,6 +1195,11 @@ static string GetItemImageKey(GitItemStatus gitItemStatus)
                     return nameof(Images.FileStatusRemoved);
                 }
 
+                if (gitItemStatus.RangeDiff)
+                {
+                    return nameof(Images.Diff);
+                }
+
```

Comment:
Could be moved to the begin of the function. Would save the check at line 1187.

Reply:
Yes, this is a leftover from supporting other info in RangeDiff

Reply:
Actually do not want to change this, the first check is for illegal combinations
If the "validility check" is considered unnecessary, FileStatusUnknown should be using the fallback at the very end.
(No other code to be changed; FileStatusAdded is a bit special but still OK.)